libraries/chain/hardfork.d/143-2.hf

```diff
@@ -0,0 +1,4 @@
+// #143 Require voted entities to exist
+#ifndef HARDFORK_143_2_TIME
+#define HARDFORK_143_2_TIME (fc::time_point_sec( 1503878400 ))
```

Comment:
Please rename to HARDFORK_CORE_143_TIME to make it clear that the number refers to the bitshares-core issue.